backend/src/govsg/services/govsg-callback.service.ts

```diff
@@ -245,7 +260,7 @@ const parseTemplateMessageWebhook = async (
         // not sure whether need to throw an error hmm probably not?
         return
       }
-      const { code, title, details, href } = body.errors[0]
+      const { code, title, details, href } = errors[0]
```

Comment:
No known good way to handle `errors` ([docs](https://developers.facebook.com/docs/whatsapp/on-premises/webhooks/components#errors-object)).

Reply:
I wrote the PR for this in a rush, but ideally we should use sth like `zod` at I/O boundaries, in this case to ensure that the shape of `errors` actually conform to the shape described in the doc. for future consideration.

Reply:
Noted on validating error shape.

This comment of mine was more of:

If there are 10 messages in `body.messages`, what about `body.errors`? None of the fields seem to help me map them to the individual message in `body.messages`.

Reply:
ohh my point was more general about mapping all I/O boundaries, not just error shape.

but ya to your point, `errors` is always length 1 also in practice.